sdk/storage/Azure.Storage.Common/src/Shared/StorageClientConfiguration.cs

```diff
@@ -15,6 +16,8 @@ internal class StorageClientConfiguration
 
         public virtual StorageSharedKeyCredential SharedKeyCredential { get; private set; }
 
+        public virtual TokenCredential OAuthTokenCredential { get; private set; }
```

Comment:
Is this new property wired up correctly on the `.Get*Client()` methods in Blobs, Files, and Data Lake?

Reply:
For now it's just on BlobClientConfigurations and not applicable towards the other services yet. We should add them when we need to.

Yes they should be wired correctly for our `Get*Client` because we pass the entire BlobClientConfigurations.